cognite_toolkit/cdf_tk/describe.py

```diff
@@ -0,0 +1,162 @@
+from __future__ import annotations
+
+import datetime
+
+from cognite.client.data_classes.aggregations import Count
+from cognite.client.data_classes.data_modeling import (
+    DirectRelation,
+    DirectRelationReference,
+)
+from rich import print
+from rich.table import Table
+
+from .utils import CDFToolConfig
+
+
+def describe_datamodel(ToolGlobals: CDFToolConfig, space_name: str, model_name: str) -> None:
+    """Describe data model from CDF"""
+
+    print(f"Describing data model ({model_name}) in space ({space_name})...")
+    print("Verifying access rights...")
+    client = ToolGlobals.verify_client(
+        capabilities={
+            "dataModelsAcl": ["READ", "WRITE"],
+            "dataModelInstancesAcl": ["READ", "WRITE"],
+        }
+    )
+    table = Table(title=f"Space {space_name}")
+    table.add_column("Info", justify="left")
+    table.add_column("Value", justify="left", style="green")
+    try:
+        space = client.data_modeling.spaces.retrieve(space_name)
+    except Exception as e:
+        print(f"Failed to retrieve space {space_name}.")
+        print(e)
+    else:
+        if space is None:
+            print(f"Failed to retrieve space {space_name}. It does not exists or you do not have access to it.")
+        else:
+            table.add_row("Name", str(space.name))
+            table.add_row("Description", str(space.description))
+            table.add_row("Created time", str(datetime.datetime.fromtimestamp(space.created_time / 1000)))
+            table.add_row("Last updated time", str(datetime.datetime.fromtimestamp(space.last_updated_time / 1000)))
+    try:
+        containers = client.data_modeling.containers.list(space=space_name, include_global=True, limit=None)
+    except Exception as e:
+        print(f"Failed to retrieve containers for data model {model_name}.")
+        print(e)
+        return
+    container_list = [(space_name, c.external_id) for c in containers.data]
```

Comment:
We have a shorthand for this `containres.as_ids()` However, I do not understand why you want to do that, as you can just iterate through the containers. See next comment